src/coreclr/utilcode/sstring.cpp

```diff
@@ -11,6 +11,7 @@
 #include "ex.h"
 #include "holder.h"
 
+#include <pal.h>
```

Comment:
I think that pal.h should have been included transitively from one of the previous includes, so this should not be needed.

Reply:
Done. Let's see if it fixes/breaks anything.